pandas/tests/io/test_parquet.py

```diff
@@ -907,6 +907,28 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    @td.skip_if_no("pyarrow")
```

Comment:
can you add a test for failure modes (e.g. an attr key or value is not a string)